RequestManagementSystem/Client/ReqClient.py

```diff
@@ -17,22 +16,20 @@
 from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.Core.DISET.RPCClient import RPCClient
 from DIRAC.Core.Utilities.List import randomize, fromChar
-from DIRAC.Core.Utilities.JEncode import strToIntDict
-from DIRAC.Core.Utilities.DEncode import ignoreEncodeWarning
 from DIRAC.ConfigurationSystem.Client import PathFinder
-from DIRAC.Core.Base.Client import Client, createClient
+from DIRAC.Core.Base.Client import Client
 from DIRAC.RequestManagementSystem.Client.Request import Request
 from DIRAC.RequestManagementSystem.private.RequestValidator import RequestValidator
 
 
-@createClient('RequestManagement/ReqManager')
```

Comment:
Why?

Reply:
Yes, sorry I had not rebased correctly...